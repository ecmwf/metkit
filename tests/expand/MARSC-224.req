retrieve,
class=ti,date=20231226/to/20231226,expver=prod,grid=0.5/0.5,levtype=sfc,padding=0,param=151,step=0/to/240/by/6,time=12,type=fc,number=all,origin=ecmf,
target="reference.OkV8i4.data"