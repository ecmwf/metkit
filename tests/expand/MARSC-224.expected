        RETRIEVE,
            CLASS      = TI,
            TYPE       = FC,
            STREAM     = OPER,
            EXPVER     = prod,
            REPRES     = SH,
            LEVTYPE    = SFC,
            PARAM      = 151,
            DATE       = 20231226,
            TIME       = 1200,
            STEP       = 0/6/12/18/24/30/36/42/48/54/60/66/72/78/84/90/96/102/108/114/120/126/132/138/144/150/156/162/168/174/180/186/192/198/204/210/216/222/228/234/240,
            NUMBER     = ALL,
            ORIGIN     = ECMF,
            TARGET     = reference.OkV8i4.data,
            RESOL      = AUTO,
            GRID       = .5/.5,
            PADDING    = 0,
            PROCESS    = LOCAL 